includes/ucf-faq-posttype.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Handles the registration of the FAQ custom post type.
+ * @author Jim Barnes
+ * @since 1.0.0
+ **/
+if ( ! class_exists( 'UCF_FAQ_PostType' ) ) {
+	class UCF_FAQ_PostType {
+		/**
+		 * Registers the FAQ custom post type.
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 **/
+		public static function register() {
+			$singular = apply_filters( 'ucf_faq_singular_label', 'FAQ' );
+			$plural = apply_filters( 'ucf_faq_plural_label', 'FAQs' );
```

Comment:
Recommend adding a `$text_domain` variable and passing it into the register function, then replacing the `'ucf_faq'` references with the `$text_domain` variable. That way if we ever need to update the text_domain, it can be done by updating just the one variable.